docs/ksh/faq.html

```diff
@@ -771,7 +771,7 @@
 <DT>How do I do global substitutions on the contents of shell variables?<DD>
 
 Use // instead of / for global substitution, ${var//aa/bb} will
-expand to the value of  with each "aa" replace by "bb".
+expand to the value of $var with each "aa" replace by "bb".
```

Comment:
This should say `the value of var` (no `$` as `$var` is the value).

Also, change `replace` to `replaced`.